node/messages/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Message types for the overseer and subsystems.
+//!
+//! Subsystems' APIs are defined separately from their implementation, leading to easier mocking.
+
+use polkadot_primitives::Hash;
+
+/// A signal used by [`Overseer`] to communicate with the [`Subsystem`]s.
+///
+/// [`Overseer`]: struct.Overseer.html
```

Comment:
```suggestion
/// [`Overseer`]: polkadot_overseer/struct.Overseer.html
```

Reply:
Does this work? `polkadot_overseer` is not a dependency of this crate. Which means it will probably not work when we only build the docs for this crate?

Reply:
Yes, i think you're right. Then the links should be removed at all I guess?